docs/workflow/blocks/reference.rst

```diff
@@ -27,3 +27,6 @@ input data items have a field called ``label`` you can omit these options.
 
 You can also set the ``required`` status of the input select list. This defaults
 to ``false``.
+
+At the moment, it does not seem possible to prefill the field with a selected value.
+There is a property called `defaultValue` which currently doesn't seems to work.
```

Comment:
It would be good to find out and clarify what is going on here in a clear way.

Reply:
Yeah, it would be good, but currently it is not the priority. So I thought of at least leaving a note for the next person.